CuratedContent/ScalingYourselfSoftwareDeveloper.md

```diff
@@ -0,0 +1,28 @@
+# Scaling Yourself as a Software Developer 
+
+<!-- deck text start -->
+Do you find yourself developing software for your main projects outside of regular work time just to look or be productive? Do interruptions from email, social media, or requests from others cause your day to slip away? This lively and very helpful video gives numerous ideas to help software developers (or anyone) to "scale" personal productivity.
```

Comment:
I would remove this first sentence in this teaser text.

Reply:
> I would remove this first sentence in this teaser text.

I agree and did.